zerver/models.py

```diff
@@ -152,6 +152,7 @@ class Realm(ModelReprMixin, models.Model):
     date_created = models.DateTimeField(default=timezone.now) # type: datetime.datetime
     notifications_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True) # type: Optional[Stream]
     deactivated = models.BooleanField(default=False) # type: bool
+    default_language = models.CharField(default=u'en', max_length=50) # type: text_type
```

Comment:
I felt the same about max_length, i went with 50 since the existing default_language field in UserProfile model also has max_length of 50.